File: Zadania/zadanie-10/playwright/auto.py
from hamulec import Hamulec
from kierownica import Kierownica
from kolo import Kolo
from silnik import Silnik

class Auto:
    def __init__(self):
        self.silnik = Silnik()
        self.kola = [Kolo(), Kolo()]
        self.hamulce = [Hamulec(), Hamulec()]
        self.kierownica = Kierownica()

    def opis(self):
        print(self.kierownica.hello())
        print(self.silnik.hello())
        for kolo in self.kola:
            print(kolo.hello())
        for hamulec in self.hamulce:
            print(hamulec.hello())


uruchom_auto = Auto()

uruchom_auto.opis()

File: Zadania/zadanie-10/java/Auto.java
package java;

public class Auto{      
    Silnik silnik;
    Kolo[] kola;
    Hamulec[] hamulce;
    Kierownica kierownica;


    Auto(){
        this.silnik = new Silnik();
        this.kola = new Kolo[] {new Kolo(), new Kolo()};
        this.hamulce = new Hamulec[] {new Hamulec() , new Hamulec()};
        this.kierownica = new Kierownica();
        }

    public void opis() {
        System.out.println(silnik.hello());
            for (Kolo kolo : kola) {
        System.out.println(kolo.hello());
            }
            for (Hamulec hamulec : hamulce) {
        System.out.println(hamulec.hello());
            }
        System.out.println(kierownica.hello());
        }
        
    public static void main(String[] args) {
        Auto mojeAuto = new Auto();
        mojeAuto.opis();
    }
}